app/views/prior_authority/applications/state_summaries/_expired.html.erb

```diff
@@ -1,4 +1 @@
-<p><strong class="govuk-tag <%= prior_authority_tag_colour(:rejected) %>"><%= t(".status") %></strong></p>
-<p><%= @application.app_store_updated_at.to_fs(:stamp) %></p>
-<br>
-<p><%= t('prior_authority.applications.show.amount_requested', amount: @primary_quote_summary.formatted_total_cost) %></p>
+<%= render "prior_authority/applications/state_summaries/sent_back" %>
```

Comment:
is this right? an `expired` partial calling a `sent_back` partial.  are we missing a generic/sharable partial perhaps?

Reply:
Yup, so "expired" is basically an extension of "sent_back" - expired is sent-back-and-then-not-actioned. So in the expired partial we want to continue showing everything we showed in the sent back state.